README.md

```diff
@@ -370,9 +373,11 @@ airbrake.AddFilter(notice =>
         return null;
 
     // clear environment variables with "token"-related keys
-    foreach (var key in notice.EnvironmentVars.Keys)
-        if (key.Contains("token"))
+    new List<string>(notice.EnvironmentVars.Keys).ForEach(key =>
```

Comment:
Example is incorrect. Would switching title from "Add documentation on NLog integration" to "Add documentation on NLog integration. Cleanup README" or something like that work here? Or using separate PRs are still the better way?